src/main/webapp/index.jsp

```diff
@@ -20,7 +20,7 @@
       <!--Site Header-->
       <header>
           <nav>
-              <div id="userpage-icon">
+              <div id="dashboard-icon-container">
               <%
               if (userLoggedIn && !userService.isUserAdmin()) { 
```

Comment:
Can you nest these two if-statements so we don't have to repeat the HTML for the User Page link?

Reply:
Yes, I'll make this change right now, thanks!